native/src/components/ThemedSearchBar.tsx

```diff
@@ -1,50 +1,62 @@
 import React, { ReactElement } from 'react'
 import { TFunction } from 'react-i18next'
-import { SearchBar } from 'react-native-elements'
-
-import { ThemeType } from 'build-configs'
+import { TextInput, View } from 'react-native'
+import styled from 'styled-components'
 
+import { CloseIcon, SearchIcon } from '../assets'
 import testID from '../testing/testID'
+import Icon from './base/Icon'
+import IconButton from './base/IconButton'
+
+const StyledBackground = styled(View)`
+  background-color: ${props => props.theme.colors.backgroundColor};
+  flex: 1;
+  height: 48px;
+  margin: 4px;
+  flex-direction: row;
+  align-items: center;
+  padding: 8px;
+  gap: 8px;
+`
+
+const StyledIcon = styled(Icon)`
+  color: ${props => props.theme.colors.textSecondaryColor};
+  height: 20px;
+`
+
+const StyledInput = styled(TextInput)`
+  padding: 4px;
+  flex: 1;
+  color: ${props => props.theme.colors.textColor};
+  font-size: 18;
```

Comment:
I am getting a warning here:
> Expected style "fontSize: 18" to contain units

Reply:
Me too, but when I add units, the size is completely ignored, so I decided to ignore the warning 🤷

Reply:
For me this has the same result, so I would strongly prefer this
```suggestion
  font-size: 18px;
```

Reply:
It did for me now too. No idea what I did wrong previously. Grrrr.